src/public/Disable-ADSIUser.ps1

```diff
@@ -91,6 +93,8 @@ function Disable-ADSIUser
                 $Account = Get-ADSIUser -Identity $Identity @ContextSplatting
                 $Account.Enabled = $false
                 $Account.Save()
+                Write-Verbose "[$FunctionName] The Account $Identity was disable"
```

Comment:
```suggestion
                Write-Verbose "[$FunctionName] The User $Identity was disabled"
```
As in the Disabke-ADSIComputer Function i would suggest to specify the Account type.
+ Fixed typo